src/app/people/page.tsx

```diff
@@ -1,5 +1,226 @@
+  gap: 25px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 40px
+    ${linearlyScaleSize({
+      minSizePx: 24,
+      maxSizePx: 96,
+      minScreenSizePx: parseInt(ScreenSize.md),
+      maxScreenSizePx: parseInt(ScreenSize.lg),
+    })};
+  padding-right: ${linearlyScaleSize({
+    minSizePx: 24,
+    maxSizePx: 8,
+    minScreenSizePx: parseInt(ScreenSize.sm),
+    maxScreenSizePx: parseInt(ScreenSize.md),
+  })};
+  width: 100%;
+  gap: 16px;
+  @media (min-width: ${ScreenSize.max}) {
+    width: ${ScreenSize.max};
+    margin: 0 auto;
+  }
+
+  & > main {
+    // make the main content (publications list) take up 85% and the Sidebar component 15%
+    width: 85%;
+  }
+`
+const SideContainer = styled.div`
+  padding-top: 96px;
+  padding-left: 30px;
+  padding-right: 30px;
+  width: 15%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  @media (max-width: ${ScreenSize.sm}) {
+    display: none;
+  }
+`
+const AlumniSectionContent = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: 1px;
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(5, 1fr);
+  }
+`
+
+const SubCategoryTitle = styled.h2`
+  ${FontVariant.title_sm}
+`
+
+const SpecialThanksSection = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 16px;
+`
+const SpecialThanksTitle = styled.h2`
+  font-size: 1.5rem;
+  color: ${Color.gray900};
+`
+
+const SpecialThanksText = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const SpecialThanksSubtitle = styled.h3`
+  ${FontVariant.body_lg}
+  color: ${Color.gray900};
+  margin: 0;
+`
+
+const SpecialThanksDescription = styled.p`
+  ${FontVariant.body_md}
+  color: ${Color.gray700};
+  gap: 4px;
+`
 
 export default function Page() {
-  return <main></main>
+  const [sidebarList, setSidebarList] = useState<string[]>([])
+  const [activeSection, setActiveSection] = useState<string | null>(null)
+  const ignoreObserver = useRef(false)
+  const sectionRefs = useRef<Record<string, HTMLElement | null>>({})
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      entries => {
+        if (ignoreObserver.current) return
+
+        entries.forEach(entry => entry.isIntersecting && setActiveSection(entry.target.id))
+      },
+      {
+        // when a section passes the area that is 10% from the top and 80% from the bottom of the viewport, it will be considered intersecting
+        rootMargin: '-20% 0px -80% 0px',
+      }
+    )
+
+    const sections = Object.values(sectionRefs.current)
+    sections.forEach(section => {
+      section && observer.observe(section)
+    })
+
+    return () => {
+      sections.forEach(section => {
+        if (section) {
+          observer.unobserve(section)
+        }
+      })
+    }
+  }, [])
+
+  useEffect(() => {
+    handleSidebarList(MEMBERS_KEY)
```

Comment:
if you call this function once at the beginning, can't we just set the initial value of the sidebarList with the logic in the handleSidebarList?

Reply:
Yes you're right. In fact, we don't even need a handleSidebarList function since the sections are fixed